remove the old ICS calendar (#11)
ansible-community/meetings: README.md

[![Discuss on Matrix at #community:ansible.com](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20on%20Matrix%20at%20%23community:ansible.com&logo=matrix)](https://matrix.to/#/#community:ansible.com)

> **NOTE - Working group calendars have moved to the [Ansible Forum Events Calendar](https://forum.ansible.com/c/events/8). In the forum, you can select which working groups you want to attend and click the `...` button to add to your personal calendar.**

Reviewer: Long-term the "move" will become irrelevant. I might do this instead:
```suggestion
** We publish working group calendars in the [Ansible Forum Events Calendar](https://forum.ansible.com/c/events/8). In the forum, you can select which working groups you want to attend and click the `...` button to add to your personal calendar.**
```
